src/main/java/seedu/taassist/logic/commands/DeletesCommand.java

```diff
@@ -19,14 +19,14 @@ public class DeletesCommand extends Command {
 
     public static final String COMMAND_WORD = "deletes";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Delete sessions for a class. "
+    public static final String MESSAGE_USAGE = "> Deletes sessions for a class.\n"
             + "Paramaters: "
             + PREFIX_SESSION + "SESSION_NAME\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_SESSION + "Lab1";
 
     public static final String MESSAGE_SUCCESS = "Sessions deleted: %s";
-    public static final String MESSAGE_SESSION_DOES_NOT_EXIST = "Session %s does not exist!";
+    public static final String MESSAGE_SESSION_DOES_NOT_EXIST = "Session [ %s ] does not exist!";
```

Comment:
This formatting might need to change if we want to allow users to delete multiple sessions at once. Specific formatting can be discussed in v1.4.